src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectDetails.tsx

```diff
@@ -32,12 +32,11 @@ const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }) => {
       : 'n/a';
   const end =
     project.workPackages.length > 0
-      ? endDatePipe(
+      ? datePipe(
           project.workPackages.reduce(
-            (min, cur) => (cur.startDate < min ? cur.startDate : min),
-            project.workPackages[0].startDate
-          ),
-          project.workPackages.reduce((tot, cur) => tot + cur.duration, 0)
+            (min, cur) => (cur.endDate < min ? cur.endDate : min),
```

Comment:
no need to reduce it anymore, you can just do date pipe on the project.endDate and then instead of the `project.workPackages.length > 0` you can just check `project.endDate` because that will return true if the date is present

Reply:
pretty sure this got resolved in a later requested change, feel free to double check for me